pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1470,6 +1470,45 @@ self: super: {
   pandoc-types = self.pandoc-types_1_21;
   rfc5051 = self.rfc5051_0_2;
 
+  # 2020-11-15: Taffybar and gi-* dependencies need these overrides to
+  # be compatible with haskell-gi-base 0.24.
+  taffybar = markUnbroken (appendPatch super.taffybar (pkgs.fetchpatch {
```

Comment:
Instead of using `markUnbroken`, you can directly edit the `hackage-packages.nix` file on the release branches.

Although you're also encouraged to mark the packages in question as unbroken in the `configuration-hackage2nix.yaml` in case someone wants to regenerate the `hackage-packages.nix` file.

Reply:
OK great - makes sense - done.